pkg/services/rendering/rendering.go

```diff
@@ -108,6 +108,14 @@ func (rs *RenderingService) RenderErrorImage(err error) (*RenderResult, error) {
 	}, nil
 }
 
+func (rs *RenderingService) RenderUnavailableImage() *RenderResult {
+	imgUrl := "public/img/rendering_plugin_not_installed.png"
+
+	return &RenderResult{
+		FilePath: filepath.Join(setting.HomePath, imgUrl),
+	}
```

Comment:
I don't think it makes sense to call it a URL:
```suggestion
	imgPath := "public/img/rendering_plugin_not_installed.png"

	return &RenderResult{
		FilePath: filepath.Join(setting.HomePath, imgPath),
	}
```